src/common/components/Layouts/LayoutWithSearch/index.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import LayoutWithSearch from '.';
+import { ContentWithPadding } from './styled';
+
+describe('Component/LayoutWithSearch test', () => {
+  it('Should match snapshots', () => {
+    const wrap = shallow(<LayoutWithSearch />);
+    expect(wrap).toMatchSnapshot();
```

Comment:
render it with your new lib would be solved that I mentioned before

Reply:
fixed at f4a6fb4f1c5a3f92abc8256fafb9511c28d0eab8